app/views/news/index.html.erb

```diff
@@ -26,7 +26,19 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
 See doc/COPYRIGHT.rdoc for more details.
 
 ++#%>
-<h2><%=l(:label_news_plural)%></h2>
+<% html_title(l(:label_news_plural)) -%>
+<%= toolbar title: l(:label_news_plural) do %>
+  <% if User.current.allowed_to?(:manage_news, @project) %>
+    <li class="toolbar-item">
+      <%= link_to new_project_news_path(@project), title: l(:label_news_new), id: 'new_news_link', class: 'button -alt-highlight' do %>
```

Comment:
Narf...

I got you wrong there. You removed the entry from the side menu and added it as a toolbar button. Now you are mimicking the behaviour that was present on the side menu item on the toolbar item.

Am I getting this right?

Not sure if I like this. In my firefox the default item that is focussed on all other pages is whatever is *before* the OP logo. That means after doing one tab, I am on the OP logo (top left). I'd liek that behaviour for the news page as well, because this seems to be consistent.